android/src/main/java/co/apptailor/googlesignin/RNGoogleSigninModule.java

```diff
@@ -79,24 +84,25 @@ public String getName() {
 
     @ReactMethod
     public void playServicesAvailable(boolean autoresolve, Promise promise) {
```

Comment:
This method is no longer called from JS.

Reply:
correct, but I think we will need it, so I think it should stay. The problem is now the JS side of things.